### PR TITLE
EXLM 1348 Article Tag Block UE & FE Changes 

### PR DESCRIPTION
**Pull request by mayukhs2904**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1348

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/
- After: https://feat-exlm-1348--exlm--adobe-experience-league.hlx.live/

### CODE REVIEW

**Review by ruhisingh1 — DISMISSED** [review]

File: blocks/article-tags/article-tags.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.article-tags-header {
+  font-family: 'Adobe Clean', sans-serif; 
+}
+
+.article-tags-view{
+  display: flex;
+  flex-direction: row;
```
Comment (noormd08) [inline blocks/article-tags/article-tags.css:7]: flex-direction: row is default. we can remove this line.

File: blocks/article-tags/article-tags.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  flex-wrap: wrap;
+}
+
+.article-tags-name{
+  font-size: 12px;
```
Comment (noormd08) [inline blocks/article-tags/article-tags.css:12]: use var(--spectrum-body-size-xs)

File: blocks/article-tags/article-tags.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  const headerDiv = htmlToElement(`
+  <div class="article-tags-header">
+    <div class="article-tags-title">
+      ${articleTagHeading.innerHTML}
+    </div>
+    <div class="article-tags-view">
+      ${createTags(solutions)}
+      ${createTags(roles)}
+      ${createTags(experienceLevels)}
+    </div>
```
Comment (noormd08) [inline blocks/article-tags/article-tags.js:30]: Please create a map here.

File: blocks/article-tags/article-tags.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  const solutions = getMetadata('coveo-solution')?.split(',').filter(Boolean);
+  const roles = getMetadata('role')?.split(',').filter(Boolean);
+  const experienceLevels = getMetadata('level')?.split(',').filter(Boolean);
+
+  const [articleTagHeading] = [...block.children].map((row) => row.firstElementChild);
+  block.innerHTML = '';
+  block.classList.add('article-tags');
+
+  const headerDiv = htmlToElement(`
+  <div class="article-tags-header">
```
Comment (noormd08) [inline blocks/article-tags/article-tags.js:22]: rename this to article-tags

**Review by devisree29 — APPROVED** [review]

File: blocks/article-tags/article-tags.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        ${roles
+          .split(',')
+          .filter(Boolean)
+          .map((value) => `<div class="article-tags-name">${value.trim()}</div>`)
+          .join('')}
+        ${experienceLevels
+          .split(',')
+          .filter(Boolean)
+          .map((value) => `<div class="article-tags-name">${value.trim()}</div>`)
+          .join('')}
```
Comment (noormd08) [inline blocks/article-tags/article-tags.js:34]: Can we merge all 3 in a map, as we are using the same class name and no parent added.

**Review by noormd08 — APPROVED** [review]

**Review by devisree29 — DISMISSED** [review]

**Pull request merged**